bin/nyc.js

```diff
@@ -42,6 +42,9 @@ if ([
   if (argv.all) nyc.addAllFiles()
 
   var env = {
+    // Support running nyc as a user without HOME (e.g. linux 'nobody'),
+    // https://github.com/istanbuljs/nyc/issues/951
+    SPAWN_WRAP_SHIM_ROOT: process.env.XDG_CACHE_HOME || require('os').homedir(),
```

Comment:
What if `SPAWN_WRAP_SHIM_ROOT` is already set?  Maybe it would be better to use:
```js
SPAWN_WRAP_SHIM_ROOT: process.env.SPAWN_WRAP_SHIM_ROOT || process.env.XDG_CACHE_HOME || require('os').homedir(),
```

Reply:
@coreyfarrell Sure, that seems fair.

I omitted this because I thought it'd be unexpected for the user's own spawn-wrap configuration (if that exists) to apply to nyc's use of spawn-wrap.

Reply:
Honoring the existing `SPAWN_WRAP_SHIM_ROOT` value prevents this from being a breaking change.